actions/queens_abilities.py

```diff
@@ -30,7 +30,8 @@ def __init__(self, ai):
                 continue
             # if not lowhp_ultralisks.closer_than(8, queen.position):
             selected = self.hatchery.closest_to(queen.position)
-            if queen.energy >= 25 and not selected.has_buff(QUEENSPAWNLARVATIMER):
+            tumors = self.ai.units.of_type([CREEPTUMORQUEEN, CREEPTUMOR, CREEPTUMORBURROWED])
```

Comment:
there is another action in `actions/units/creep_tumors.py`  which does handle the tumors, no need to handle them twice

Reply:
i added this so that the first 25 queen energy of the game always go to a new tumor, as one single early tumor will greatly improve creep spread

Reply:
alright got it, use `self.ai.tumors` for that then

Reply:
pls test adding self.ai.tumors, i think it didnt work for me